.changesets/maint_garypen_rhai_update.md

```diff
@@ -0,0 +1,14 @@
+### chore: Update rhai to latest release (1.19.0) ([PR #5655](https://github.com/apollographql/router/pull/5655))
+
+In Rhai 1.18.0, there were changes to how exceptions within functions were created. For details see: https://github.com/rhaiscript/rhai/blob/7e0ac9d3f4da9c892ed35a211f67553a0b451218/CHANGELOG.md?plain=1#L12
+
+We've modified how we handle errors raised by Rhai to comply with this change, which means error message output is affected. The change means that errors in functions will no longer document which function the error occurred in, for example:
+
+```diff
+-         "rhai execution error: 'Runtime error: I have raised an error (line 223, position 5)\nin call to function 'process_subgraph_response_string''"
++         "rhai execution error: 'Runtime error: I have raised an error (line 223, position 5)'"
+```
```

Comment:
That sounds like Rhai (itself) made the user experience a bit less helpful. 😢    Is there a way to still capture this ourselves and make it available to the user?  I suppose you can still just go to the line number, but that's a small step backwards in my mind (Again, not our fault and probably more important to update.)

Reply:
Looks like we can't.  Oh well.  That's unfortunate.